controllers/clusterversion_controller.go

```diff
@@ -68,18 +75,6 @@ func (r *ClusterVersionReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *ClusterVersionReconciler) SetupWithManager(mgr ctrl.Manager) error {
-	err := mgr.Add(manager.RunnableFunc(func(context.Context) error {
-		clusterVersion, err := util.DetermineOpenShiftVersion(r.Client)
-		if err != nil {
-			return err
-		}
-
-		return r.ensureConsolePlugin(clusterVersion)
-	}))
-	if err != nil {
-		return err
-	}
-
```

Comment:
@bipuladh u remember why this custom runnable was added ?? Removing it seems reasonable, but double checking.